lib/containers/bats.pm

```diff
@@ -0,0 +1,71 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Common functions for BATS test suites
+# Maintainer: qa-c@suse.de
+
+package containers::bats;
+
+use base Exporter;
+use Exporter;
+
+use base "consoletest";
+use testapi;
+use utils;
+use strict;
+use warnings;
+use version_utils;
+use registration qw(add_suseconnect_product get_addon_fullname);
+use transactional qw(trup_call check_reboot_changes);
+
+our @EXPORT = qw(install_bats add_packagehub remove_mounts_conf switch_to_user);
+
+sub install_bats {
+    my $bats_version = "1.11.0";
+
+    script_retry("curl -sL https://github.com/bats-core/bats-core/archive/refs/tags/v$bats_version.tar.gz | tar -zxf -", retry => 5, delay => 60, timeout => 300);
+    assert_script_run "cd bats-core-$bats_version";
+    assert_script_run "bash ./install.sh /usr/local";
+}
+
+sub add_packagehub {
```

Comment:
I belive this is a subroutine that doesn't belong into `containers/bats.pm` but at a higher level.

Reply:
> I belive this is a subroutine that doesn't belong into `containers/bats.pm` but at a higher level.

Yes.  I suggest we do it in another PR when we use it in another module.